<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>met3</class>
 <widget class="QMainWindow" name="met3">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>674</width>
    <height>211</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>met3</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>30</y>
      <width>61</width>
      <height>112</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QDoubleSpinBox" name="doubleSpinBox_20"/>
     </item>
     <item>
      <widget class="QDoubleSpinBox" name="doubleSpinBox_19"/>
     </item>
     <item>
      <widget class="QDoubleSpinBox" name="doubleSpinBox_18"/>
     </item>
     <item>
      <widget class="QDoubleSpinBox" name="doubleSpinBox_17"/>
     </item>
    </layout>
   </widget>
   <widget class="QTextBrowser" name="textBrowser">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>30</y>
      <width>211</width>
      <height>61</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>90</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>=</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>10</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>Matriz 4x4</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_19">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>80</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>=</string>
    </property>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>30</y>
      <width>330</width>
      <height>120</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <widget class="QDoubleSpinBox" name="doubleSpinBox_13"/>
       </item>
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>X1</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDoubleSpinBox" name="doubleSpinBox_14"/>
       </item>
       <item>
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string>X2</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDoubleSpinBox" name="doubleSpinBox_15"/>
       </item>
       <item>
        <widget class="QLabel" name="label_12">
         <property name="text">
          <string>X3</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDoubleSpinBox" name="doubleSpinBox_16"/>
       </item>
       <item>
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>X4</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QDoubleSpinBox" name="doubleSpinBox_4"/>
       </item>
       <item>
        <widget class="QLabel" name="label_16">
         <property name="text">
          <string>X1</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDoubleSpinBox" name="doubleSpinBox_3"/>
       </item>
       <item>
        <widget class="QLabel" name="label_10">
         <property name="text">
          <string>X2</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDoubleSpinBox" name="doubleSpinBox_2"/>
       </item>
       <item>
        <widget class="QLabel" name="label_9">
         <property name="text">
          <string>X3</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDoubleSpinBox" name="doubleSpinBox"/>
       </item>
       <item>
        <widget class="QLabel" name="label_14">
         <property name="text">
          <string>X4</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QDoubleSpinBox" name="doubleSpinBox_9"/>
       </item>
       <item>
        <widget class="QLabel" name="label_18">
         <property name="text">
          <string>X1</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDoubleSpinBox" name="doubleSpinBox_10"/>
       </item>
       <item>
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>X2</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDoubleSpinBox" name="doubleSpinBox_11"/>
       </item>
       <item>
        <widget class="QLabel" name="label_15">
         <property name="text">
          <string>X3</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDoubleSpinBox" name="doubleSpinBox_12"/>
       </item>
       <item>
        <widget class="QLabel" name="label_8">
         <property name="text">
          <string>X4</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QDoubleSpinBox" name="doubleSpinBox_6"/>
       </item>
       <item>
        <widget class="QLabel" name="label_17">
         <property name="text">
          <string>X1</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDoubleSpinBox" name="doubleSpinBox_5"/>
       </item>
       <item>
        <widget class="QLabel" name="label_13">
         <property name="text">
          <string>X2</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDoubleSpinBox" name="doubleSpinBox_7"/>
       </item>
       <item>
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>X3</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDoubleSpinBox" name="doubleSpinBox_8"/>
       </item>
       <item>
        <widget class="QLabel" name="label_11">
         <property name="text">
          <string>X4</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>120</y>
      <width>254</width>
      <height>23</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QPushButton" name="pushButton">
       <property name="text">
        <string>METODO</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_2">
       <property name="text">
        <string>BORRA</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_3">
       <property name="text">
        <string>SAL</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>674</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
